mlflow/models/evaluation/base.py

```diff
@@ -12,7 +12,7 @@
 from contextlib import contextmanager
```

Comment:
Can we add test coverage for this change to `test_default_evaluator.py`?

Reply:
Well the default evaluator always invokes the model with pandas dataframe so we cannot test this functionality with the default evaluator. Instead, I'll add unit tests for the `ModelFromDeploymentEndpoint` class in `test_evaluation.py`.

Reply:
Even so, can we add tests to `test_default_evaluator.py` verifying that a user can pass a list of chat histories as an input, rather than a dataframe?

Reply:
@dbczumar  It's not a valid input for the `mlflow.evaluate()`. The dictionary input we are taking here is what passed by `evaluator` to `PythonModel`, not provided by the users.

Reply:
Got it - thanks :)